Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -279,6 +265,11 @@ - (void)setupToolbar {
 }
 
 - (void)updateToolbarItemsIfNeeded {
+    
+    if (!self.saveButtonController) {
```

Comment:
@bgerstle because then you would have to call [self saveButtonController] just for the side effect on creating it. And that seemed worse.